aiogram/utils/text_decorations.py

```diff
@@ -57,14 +57,14 @@ def unparse(self, text: str, entities: Optional[List[MessageEntity]] = None) ->
         """
         result = "".join(
             self._unparse_entities(
-                text, sorted(entities, key=lambda item: item.offset) if entities else []
+                text.encode('utf-16-le'), sorted(entities, key=lambda item: item.offset) if entities else []
```

Comment:
Small optimization.

Please move calling of encode to separated method:
```python3
def _add_surrogates(self, text: str) -> bytes:
    return text.encode('utf-16-le')
``` 
And also use adding/removing surrogates inside `_unparse_entities` method